packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -755,8 +761,8 @@ $side-padding: 1.2em;
         /* icon refId currency buy_price potential_payout potential_profit indicative_price rem_time */
```

Comment:
Would you please update the comment here with the new column?

Reply:
Ok, I updated the comment